packages/wp-comments/src/index.ts

```diff
@@ -97,6 +98,16 @@ export const insertComment = (
   }
 };
 
+/**
+ * Checks if a data object represents a list of comments.
+ *
+ * @param data - Object of type {@link Data}.
+ * @returns A boolean value with the result.
+ */
+export function isComments(data: Data): data is CommentData {
```

Comment:
I wonder which one is correct:

- `isComments()`
- `areComments()`

Have we already talked about this?

Reply:
My opinion is that it's fine if we consider "Comments" to be a proper noun like "Rolling Stones". 
Syntactically, `areComments` is more appropriate, but I think`isComments` is expressing the semantics better and it's also more consistent with the other type guards.

It's like asking "is this thing (Data) in the category of Comments".

Reply:
Thanks for your input Michal! 🙂 

I have the feeling that we have already talked about this, haven't we?

Reply:
yes 😅 I think it was in the FD 🙂

Reply:
I thought so as well, but I cannot find any reference to `isComments` there: https://community.frontity.org/t/wordpress-comments-package/1267

@darerodz, do you know where is it?

Reply:
I remember the conversation but I wasn't able to find it either.

Reply:
Ok, @mburridge has confirmed that `isComments` sounds fine and seems correct.

Sorry about this. I didn't want to risk releasing something that could sound really wrong for a native English speaker 😅

I guess we can follow rule this from now on 🙂